src/html/htmlcell.cpp

```diff
@@ -65,7 +65,10 @@ wxColour
 wxDefaultHtmlRenderingStyle::
 GetSelectedTextBgColour(const wxColour& WXUNUSED(clr))
 {
-    return wxSystemSettings::GetColour(wxSYS_COLOUR_HIGHLIGHT);
+    wxSystemColour colourIndex = wxSYS_COLOUR_HIGHLIGHT;
```

Comment:
Very minor, but why initialize it to a value only to overwrite it in the next line?

I'd write just `!m_wnd || m_wnd->HasFocus() ? wxSYS_COLOUR_HIGHLIGHT : wxSYS_COLOR_BTNSHADOW`.

I'd also add a comment:

    // By default we use the fixed standard selection colour, but if we're associated with a window
    // use the colour appropriate for the window state, i.e. grey out selection when it's not in focus.